test/spec_lint.rb

```diff
@@ -225,6 +225,11 @@ def obj.fatal(*) end
     }.must_raise(Rack::Lint::LintError).
       message.must_match(/must start with/)
 
+    # XXX not actually sure what *should* happen in an OPTIONS * request.
```

Comment:
What should we do here?

Reply:
If you do an `OPTIONS *` it seems to set `PATH_INFO` to `*` and not touch `SCRIPT_NAME` which I suppose is fine but there might be implications e.g. for things like `Rack::Request#path`.